tests/core/test_subscription_observer.py

```diff
@@ -34,7 +34,7 @@ def test_subscription_observer_compare():
     label = Label()
```

Comment:
This whole test can go away. The only thing we care about is pointer identity, which is different for two different objects, by definition.